### DISCUSSION TITLE
I wrote that blog post on how to 0 + 1.

### DISCUSSION THREAD

**Discussion by stevenhuyn** [zicklag/blog#5] [General]
If you think it is ok, feel free to link it.

https://stevenhuyn.bearblog.dev/succ/

**Comment by zicklag** [comment]
Thanks! I added an extra section to the bottom of the post linking to yours! Thanks for continuing the investigation!

**Comment by KaneRoot** [comment]
Hello, I read your blog post some time ago but it seems offline now. Is there any way to access it? Thanks

**Comment by stevenhuyn** [comment]
oh shoot my bad let me get it back on

**Reply by stevenhuyn** [reply]
it's back, but safer to use this link:

https://web.archive.org/web/20250118191241/https://stevenhuyn.bearblog.dev/succ/

**Reply by KaneRoot** [reply]
Thanks for your time.
